backend/pluto/pluto/_internal/server/flask_server.py

```diff
@@ -52,17 +57,56 @@ def configs(self, config: Config):
             self.app.config[config_name.upper()] = value
 
     def _add_endpoints(self):
+        self._add_expense_endpoints()
+        self._add_income_endpoints()
+        self._add_user_endpoints()
+
+    def _add_expense_endpoints(self):
+        self.add_endpoint(
+            "/expenses/",
+            "list_expense",
+            self.list_expense,
+            methods=["GET"],
+        )
         self.add_endpoint(
             "/expenses/", "add_expenses", self.add_expense, methods=["POST"]
         )
-        self.add_endpoint("/incomes/", "add_income", self.add_income, methods=["POST"])
         self.add_endpoint(
             "/upload/expenses/",
             "upload_expense_file",
             self.expense_file_upload,
             methods=["POST", "GET"],
         )
 
+    def _add_income_endpoints(self):
+        self.add_endpoint(
+            "/incomes/",
+            "list_income",
+            self.list_income,
+            methods=["GET"],
+        )
+        self.add_endpoint("/incomes/", "add_income", self.add_income, methods=["POST"])
+        self.add_endpoint(
+            "/upload/incomes/",
+            "upload_income_file",
+            self.income_file_upload,
+            methods=["POST", "GET"],
+        )
+
+    def _add_user_endpoints(self):
+        self.add_endpoint(
```

Comment:
Dois métodos para a mesma rota `/users/`mas para métodos diferentes. Isso funciona mesmo?

Reply:
It works :) . That's the usual behavior of HTTP APIs. It's just a single HTTP resource with two allowed methods.